packages/client/package.json

```diff
@@ -44,5 +44,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
lets move this to `dependencies` instead, `devDependencies` are for packages that only get used in development (like testing/linting) and can be skipped when deploying!

can do this by uninstalling (`npm uninstall @material-ui/core`) and reinstalling (`npm install @material-ui/core`), it should automatically get added again under `dependencies`